controllers/servers/host_definer/common/types.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class DefineHostRequest:
+    prefix: str = field(default_factory=str)
+    secret_name: str = field(default_factory=str)
+    secret_namespace: str = field(default_factory=str)
```

Comment:
the servicer and mediator dont need it, please consider moving or splitting it..